Mu2eG4/src/constructSaddles.cc

```diff
@@ -314,7 +314,7 @@ namespace mu2e {
 	
 	} // end of for loop over saddles
     
-    } else { // end of version 1, 2.  At this time, other versions are not
+    } else { // end of version 1, 2, 3.  At this time, other versions are not
       // implemented.
```

Comment:
The printout needs to be updated (add 3); Use G4cout, G4endl in code called by Geant4 such as this one